Dockerfile

```diff
@@ -56,6 +57,13 @@ USER valhalla
 
 WORKDIR /custom_files
 
+# Smoke tests
+RUN    python3 -c "import valhalla,sys; print (sys.version, valhalla)" \
+    && valhalla_build_config | jq type \
+    && valhalla_build_tiles -v \
+    && cat /usr/local/src/valhalla_version \
```

Comment:
this is new to me :sweat_smile: are there more things in that directory? 

can we remove that line or the one above, since they both seem to print the version?

Reply:
talking about line 64 here btw..